includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -892,6 +887,7 @@ public function display_payment_request_button_html() {
 			</div>
 		</div>
 		<?php
+		$this->display_payment_request_button_separator_html();
```

Comment:
Note that I've moved the separator to be called directly after we print out the button container. Before this PR the button and separator were on the same hook but different priorities. That leads to issues if other gateways hook onto the same priority as our button. eg:

<img width="337" alt="image" src="https://github.com/woocommerce/woocommerce-gateway-stripe/assets/8490476/dff2808a-7872-4fe8-ab01-73fcc1a623fb">

Both buttons are on priority 1, **- OR -** is on 2.